mods/parsoid.js

```diff
@@ -115,19 +134,39 @@ PSP.saveParsoidResult = function (restbase, req, format, tid, parsoidResp) {
     }
 };
 
-PSP.generateAndSave = function(restbase, req, format, tid) {
+// Temporary work-around for Parsoid issue
+// https://phabricator.wikimedia.org/T93715
+function normalizeHtml(html) {
+    return html && html.toString
+        && html.toString().replace(/ about="[^"]+"(?=[\/> ])/g, '');
+}
+function sameHtml(a, b) {
+    return normalizeHtml(a) === normalizeHtml(b);
+}
+
+PSP.generateAndSave = function(restbase, req, format, currentContentRes) {
     var self = this;
-    if (!tid) {
-        throw new Error('no tid');
-    }
+    var tid = uuid.v1();
     // Try to generate HTML on the fly by calling Parsoid
     var rp = req.params;
+    var storageRequest = null;
+
     return restbase.get({
         uri: new URI([rp.domain,'sys','parsoid','pagebundle',
                      normalizeTitle(rp.title),rp.revision])
     })
-    .then(function(parsoidResp) {
-        return self.saveParsoidResult(restbase, req, format, tid, parsoidResp);
+    .then(function(res) {
```

Comment:
If we get rid of `/sys/parsoid/pagebundle/` the `get()` above needs to be turned into a method call